influxdb/tests/py26_test.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
```

Comment:
You should change this to `#!/usr/bin/env python` otherwise this might not use the correct version.